packages/lesswrong/components/shortform/ShortformSubmitForm.jsx

```diff
@@ -9,6 +9,21 @@ const styles = theme => ({
   }
 }) 
 
+const forumHintText = {
+  LessWrong: <div>
+    <div>Write your thoughts here! What have you been thinking about?</div>
+    <div>Exploratory, draft-stage, rough, and rambly thoughts are all welcome on Shortform.</div>
+  </div>,
+  AlignmentForum: <div>
+    <div>Write your thoughts here! What have you been thinking about?</div>
+    <div>Exploratory, draft-stage, rough, and rambly thoughts are all welcome on Shortform.</div>
+  </div>,
+  EAForum: <div>
+    <div>Write your brief or quickly written post here.</div>
+    <div>Exploratory, draft-stage, rough, and off-the-cuff thoughts are all welcome on Shortform.</div>
```

Comment:
FWIW, I think it makes slightly more sense to just remove "rambly" rather than replace it with off-the-cuff, which seems to mean something similar to rough/draft-stage and exploratory. But YMMV.

Reply:
Aaron will come through with his own copy edits. This is my first pass.